webapp/channels/src/components/dialog_router/interactive_dialog_adapter.tsx

```diff
@@ -178,9 +178,9 @@ class InteractiveDialogAdapter extends React.PureComponent<Props> {
                 callback_id: this.props.callbackId || '',
                 state: this.props.state || '',
                 submission: convertedValues as {[x: string]: string},
-                user_id: '',
-                channel_id: '',
-                team_id: '',
+                user_id: '', // Populated by submitInteractiveDialog action
+                channel_id: '', // Populated by submitInteractiveDialog action
+                team_id: '', // Populated by submitInteractiveDialog action
```

Comment:
We don't have access to Redux here, so handle in the action.
Cannot leave null, document they are filled in by submitInteractiveDialog.